l5kit/l5kit/data/map_api.py

```diff
@@ -176,6 +272,21 @@ def get_crosswalk_coords(self, element_id: str) -> dict:
 
         return {"xyz": xyz}
 
+    def is_traffic_light(self, element_id: str) -> bool:
+        """
+        Check if the element is a traffic light
+        Args:
+            element_id (str): the id (utf-8 encode) of the element
+
+        Returns:
+            True if the element is a traffic light
+        """
+        element = self[element_id]
+        if not element.element.HasField("traffic_control_element"):
+            return False
+        traffic_el = element.element.traffic_control_element
+        return traffic_el.HasField("traffic_light") is True
+
     def is_traffic_face_colour(self, element_id: str, colour: str) -> bool:
```

Comment:
```suggestion
    def check_traffic_light_state(self, element_id: str, tl_state: str) -> bool:
```

Reply:
state is something like `ACTIVE, INACTIVE, here we're just checking if the element is a traffic light face and which colour it has. It's a static info, not a dynamic one